db.php

```diff
@@ -0,0 +1,10 @@
+<?php
+  $servername = 'localhost';
+  $username = 'root';
+  $password = '';
+  $dbname = "my_db";
```

Comment:
Use php constants to store these.
https://www.php.net/manual/en/language.constants.php